pkg/apps/installer.go

```diff
@@ -235,9 +236,24 @@ func (i *Installer) Poll() (man *Manifest, done bool, err error) {
 }
 
 func updateManifest(db couchdb.Database, man *Manifest) error {
-	return couchdb.UpdateDoc(db, man)
+
+	if err := permissions.Destoy(db, man.Slug); err != nil {
+		return err
+	}
+
+	if err := couchdb.UpdateDoc(db, man); err != nil {
+		return err
+	}
+
+	return permissions.Create(db, man.Slug)
```

Comment:
When we will have optional permissions, we will have to do something more complex here to preserve them when an app is updated. But for now, it's OK.